README.md

```diff
@@ -7,6 +7,27 @@ This is the development repository for the uDALES (urban Dutch Atmospheric Large
 <img src="docs/assets/images/fielddump_slice_3D.102.png" alt="uDALES" height="256"> 
 </div>
 
+## Overview
+
+With continuing urbanization, challenges associated with the urban environment such as air quality, heat islands, pedestrian thermal comfort, and wind loads on tall buildings, are increasingly relevant. Our ability to realistically capture processes such as the transport of heat, moisture, momentum and pollutants, and those of radiative transfer in urban environments is key to understanding and facing these challenges (Oke et al., 2017).
+
+The turbulent nature of the urban flow field and the inherent heterogeneity and wide range of scales associated with the urban environment result in a complex modelling problem. Large-eddy simulation (LES) is an approach to turbulence modelling used in computational fluid dynamics to simulate turbulent flows over a wide range of spatial and temporal scales. LES is one of the most promising tools to model the interactions typical of urban areas due to its ability to resolve the urban flow field at resolutions of _O_(1 m, 0.1 s), over spatial domains of _O_(100 m), and time periods of _O_(1 h). Although there are many scalable LES models for atmospheric flows, to our knowledge, only few are capable of explicitly representing buildings and of modelling the full range of urban processes (e.g. PALM-4U; Resler et al. 2017; Maronga et al. 2020; or OpenFoam; Weller et al. 1998)..
```

Comment:
I would maybe add a sentence like "uDALES is a large-eddy simulation model for urban environments" before saying: Large-eddy simulation is an approach to turbulence modelling ... and maybe remove the last sentence where we reference other models, as this is not really important for the code repository description.

Reply:
I have added the suggested first sentence.

I think we want to keep in the last sentence of that paragraph as there is a need to show a clear statement of need in the documentation (#137) and I think this point is key to understanding what uDALES offers to the field.